src/pages/MainPage/MenuBottom.jsx

```diff
@@ -1,125 +1,204 @@
+import { Link } from 'react-router-dom';
+
 const MenuBottom = () => {
   return (
     <section className="min-h-80 py-16">
       <div className="container mx-auto">
         <div className="mx-auto flex justify-between border-b border-t border-dark-color px-10 py-8">
           <div className="font-fourth">
             <h3 className="text-xl/3 font-semibold">
-              <a href="#" className="text-dark-color">
+              <Link
+                to="/cities-guides"
+                className="text-dark-color visited:text-purple-900"
```

Comment:
For the future improvements:

`MenuBottom` has a lot of repetitive code like this

```jsx
                <Link
                  className="visited:text-purple-900"
                  to="/cities-guides/id"
                >
                  Paris
                </Link>
```

You can create a more granular component to avoid repetitions

Reply:
MenuBottomLink(props) component was added